name-registry/project/contracts/registry-contract/tests/utils/interface/core.rs

```diff
@@ -1,29 +1,23 @@
 use crate::utils::setup::{get_timestamp_and_call, NameRegistry};
 use fuels::{
-    prelude::{CallParameters, ContractId, TxParameters, WalletUnlocked},
+    prelude::{AssetId, CallParameters, TxPolicies, WalletUnlocked},
     programs::call_response::FuelCallResponse,
-    types::{AssetId, Identity, SizedAsciiString},
+    types::Identity,
 };
 
 pub(crate) async fn extend(
     instance: &NameRegistry<WalletUnlocked>,
-    name: &String,
+    name: String,
```

Comment:
Just noting that I think we could use string slices here for more flexibility but it doesn't really matter

Reply:
This is using the fuels-rs `String` type